src/sentry/tsdb/snuba.py

```diff
@@ -602,6 +604,7 @@ def __get_data_legacy(
                 referrer=f"tsdb-modelid:{model.value}",
                 is_grouprelease=(model == TSDBModel.frequent_releases_by_group),
                 use_cache=use_cache,
+                tenant_ids=tenant_ids,
```

Comment:
nit: i probably wouldn't declare a variable for `tenant_ids` if it were only used once and doesn't really impact legibility, but this is a matter of taste and is fine as is. (this is called inline variable in martin fowler's _refactoring_)